# include prototypes for musl
# should not be neccesary for keepassxc 2.3.2
# see: https://github.com/keepassxreboot/keepassxc/pull/1726

--- src/sshagent/blf.h
+++ src/sshagent/blf.h
@@ -30,6 +30,8 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
+#define _GNU_SOURCE
+#include <sys/types.h>
 
 #ifndef _BLF_H_
 #define _BLF_H_
--- src/sshagent/blowfish.c
+++ src/sshagent/blowfish.c
@@ -49,7 +49,6 @@
 #include <string.h>
 #endif
 
-#include <sys/types.h>
 #ifdef HAVE_BLF_H
 #include "blf.h"
 #endif
